src/poly.rs

```diff
@@ -285,7 +285,7 @@ impl<'a, F: Field, B: Basis> Mul<F> for Polynomial<F, B> {
 /// Describes the relative rotation of a vector. Negative numbers represent
 /// reverse (leftmost) rotations and positive numbers represent forward (rightmost)
 /// rotations. Zero represents no rotation.
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
I'm a _little_ worried about comparisons between `Rotation`s because for some circuits `Rotation(x)` is the same "logical" rotation as `Rotation(y)` even if `x != y`.

Reply:
This is here because the dev tooling uses it in a `BTreeMap`, but we've established a pattern of having separate copies in `crate::dev::metadata`, so we could do the same here.